src/conic_form.jl

```diff
@@ -26,14 +26,14 @@ _set_type(::MOI.ConstraintIndex{F,S}) where {F,S} = S
 cons_offset(conic::MOI.Zeros) = conic.dimension
 cons_offset(conic::MOI.Nonnegatives) = conic.dimension
 cons_offset(conic::MOI.SecondOrderCone) = conic.dimension
-cons_offset(conic::MOI.PositiveSemidefiniteConeTriangle) = Int64((conic.side_dimension*(conic.side_dimension+1))/2)
+cons_offset(conic::MOI.PositiveSemidefiniteConeTriangle) = MOI.dimension(conic)
```

Comment:
All `cons_offset` method are equivalent to `MOI.dimension` so we should just remove this `cons_offset` function.

Reply:
Missed your other comment, let's do it in a separate PR indeed